x-pack/metricbeat/module/stan/_meta/kibana/7/dashboard/Metricbeat-STAN-overview.json

```diff
@@ -9,7 +9,7 @@
             "filter": [],
             "query": {
               "language": "kuery",
-              "query": "STAN Metricbeats"
+              "query": ""
```

Comment:
Have no idea how this got in here. I don't recall doing a global string replace.

Reply:
👍